src/extras/SceneUtils.js

```diff
@@ -18,21 +18,46 @@ THREE.SceneUtils = {
 
 	},
 
-	detach: function ( child, parent, scene ) {
+	detach: function ( child, scene ) {
+
+		var topParent = child.parent;
+		
+		while ( topParent.parent !== undefined ) {
+			
+			topParent = topParent.parent;
+			
+		}
+		
+		topParent.updateMatrixWorld( true );
```

Comment:
Yes, @WestLangley just pointed this out :(

Sensible to drop parent as parameter though? Else you may be trying to remove from an unrelated item